src/lib/familiar.ts

```diff
@@ -1,5 +1,16 @@
-import { Familiar } from "kolmafia";
-import { $familiar, $item, have } from "libram";
+import {Familiar, familiarWeight, inebrietyLimit, myFamiliar, myInebriety, numericModifier} from "kolmafia";
+import { $familiar, $item, have, get } from "libram";
+
+export function selectBestFamiliar(): Familiar {
+  if( myFamiliar() === $familiar`Stooper` && myInebriety() === inebrietyLimit() || myInebriety() === inebrietyLimit() + 1) {
```

Comment:
I don't believe we need stooper when `myInebriety() === inebrietyLimit()`

Reply:
When stooper is already out, we need to keep it, or else we can't adventure anymore:
```> ash inebriety_limit()

Returned: 15

> familiar stooper; ash inebriety_limit()

Putting Shadows Dance in the Darkness the Disgeist back into terrarium...
Taking To Binge the Stooper out of terrarium...
amulet coin is better than (none). Switching items...
Putting on amulet coin...
Equipment changed.
Returned: 16```

Reply:
:+1: